content/documentation/api.md

```diff
@@ -26,6 +26,10 @@ numbers. If `xs` is empty, return 1.
 ```
 Return `a` to the power of `x`.
 
+## `*compile-mode*`
+
+
+
```

Comment:
There is no doc for this? 🤔 I will double check that, maybe for the next version we should add some information about this.

Reply:
<img width="1300" alt="Screenshot 2021-12-31 at 13 04 30" src="https://user-images.githubusercontent.com/5256287/147822427-edc9edde-01ca-4189-a3be-426b3e804320.png">

Interesting 🧐 the doc is there in the code, but when I run `(doc *compile-mode*)` there is no doc?
Any ideas about it, @jenshaase?

Reply:
I think I know the problem: `doc` looks in `$GLOBALS` for the documentation. But not in GlobalEnvironment class. I will not fix it now. There needs to be a bigger refactoring of GlobalEnvironment and $GLOBALS in future.